linux/LinuxMDEparser/README.MD

```diff
@@ -0,0 +1,51 @@
+# LinuxMDEparser
+Microsoft Defender Endpoint Log Parser for Linux.
+
+# About
+The LinuxMDEparser is a Windows tool that allows you to export or parse MDE for Linux log files to Excel, making it easier to filter and search the logs. After collecting the MDE logs from any Linux machine, you can then run this tool on your on Windows.
```

Comment:
Please remove references to Windows as the only supported platform, as from what I can see nothing in your code requires Windows. This repository is for cross-platform Defender. Mandating Windows for handling non-Windows product raises too many questions/do not look right for customers who use no Windows in their daily work.

It is totally OK to include instructions on how to run it on Windows, complementing instructions on how to run the tool from bare Python.

Reply:
Committed as requested